### PR TITLE
Extended authentication error log message.

### PR DESCRIPTION
**Pull request by peterserfozo**

### CODE REVIEW

**Review by mxr576 — APPROVED** [review]

**Comment by mxr576** [comment]
Let's hold back with merging this until this solution gets confirmed by @cnovak in the JIRA user story.

**Review by tamasd — APPROVED** [review]

File: config/schema/apigee_edge.schema.yml [hunk 1]
```diff
@@ -178,3 +178,23 @@ apigee_edge.dangerzone:
...
+      label: 'Placeholder for a Callback URL field'
+    callback_url_pattern:
+      type: string
+      label: 'Regular expression that a Callback URL should match'
+    callback_url_description:
+      type: string
+      label: 'Description of a Callback URL field'
+    callback_url_pattern_error_message:
+      type: string
+      label: 'Client-side validation error message if a Callback URL does not match'
```
Comment (mxr576) [inline config/schema/apigee_edge.schema.yml]: Missing new line in the end of the file.

File: tests/src/Functional/EdgeAuthenticationTest.php [hunk 2]
```diff
@@ -29,6 +29,13 @@
  */
 class EdgeAuthenticationTest extends ApigeeEdgeFunctionalTestBase {
 
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'dblog',
```
Comment (mxr576) [inline tests/src/Functional/EdgeAuthenticationTest.php]: Why it is required?

File: tests/src/Functional/ApigeeEdgeTestTrait.php [hunk 3]
```diff
@@ -37,9 +37,7 @@
    * {@inheritdoc}
    */
   protected function setUp() {
-    self::$modules = [
-      'apigee_edge_test',
-    ];
+    self::$modules[] = 'apigee_edge_test';
```
Comment (mxr576) [inline tests/src/Functional/ApigeeEdgeTestTrait.php]: Use the new installExtraModules() method instead.

File: src/Form/AuthenticationForm.php [hunk 4]
```diff
@@ -367,9 +380,15 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
     }
     catch (\Exception $exception) {
       watchdog_exception('apigee_edge', $exception);
-      $form_state->setError($form, $this->t('Connection failed. Response from edge: %response', [
+      $message = 'Connection failed. Response from Apigee Edge: %response';
```
Comment (mxr576) [inline src/Form/AuthenticationForm.php]: The string is p~1 not translatable this way.

**Review by mxr576 — CHANGES_REQUESTED** [review]

**Review by mxr576 — APPROVED** [review]

**Pull request merged**